From 6e7ad272d08767455aef658066b981f513ad0cec Mon Sep 17 00:00:00 2001
From: Aaron Shaw <shawaj@gmail.com>
Date: Tue, 10 Jan 2017 16:05:41 +0000
Subject: [PATCH] Add driver_name property

Add driver name property for use with 5.1 passthrough audio in LibreElec and other Kodi based OSs
---
 sound/soc/bcm/justboom-dac.c | 1 +
 1 file changed, 1 insertion(+)

--- a/sound/soc/bcm/justboom-dac.c
+++ b/sound/soc/bcm/justboom-dac.c
@@ -98,6 +98,7 @@ static struct snd_soc_dai_link snd_rpi_j
 /* audio machine driver */
 static struct snd_soc_card snd_rpi_justboom_dac = {
 	.name         = "snd_rpi_justboom_dac",
+	.driver_name  = "JustBoomDac",
 	.owner        = THIS_MODULE,
 	.dai_link     = snd_rpi_justboom_dac_dai,
 	.num_links    = ARRAY_SIZE(snd_rpi_justboom_dac_dai),
